src/dscontrib/gkaberere/nonDesktop/weeklyActivation/day_2_7_activation.sql

```diff
@@ -0,0 +1,79 @@
+DECLARE cohort_date DATE DEFAULT '2020-03-01';
+DECLARE activation_period_start DATE DEFAULT '2020-03-02';
+DECLARE activation_period_end DATE DEFAULT '2020-03-07';
+/*DECLARE os_name STRING DEFAULT 'Android';
+DECLARE app STRING DEFAULT 'Fennec';*/
+
+WITH last_seen_data as (
+SELECT
+  submission_date,
+  client_id,
+  days_since_seen,
+  days_since_created_profile,
+  profile_date,
+  app_name,
+  SPLIT(metadata_app_version,'.')[offset(0)] as app_version,
+  os,
+  normalized_channel,
+  campaign,
+  country,
+  distribution_id,
+  CASE app_name
+    WHEN 'Fennec' THEN CONCAT(app_name, ' ', os)
+    WHEN 'Focus' THEN CONCAT(app_name, ' ', os)
+    WHEN 'Lockbox' THEN CONCAT('Lockwise ', os)
+    WHEN 'Zerda' THEN 'Firefox Lite'
+    ELSE app_name
+  END AS product_name
+FROM
+  `moz-fx-data-shared-prod.telemetry.core_clients_last_seen`
+WHERE
+  submission_date >= cohort_date
+  AND submission_date <= activation_period_end
+  /*AND app_name = app
+  AND os = os_name*/
+  AND days_since_seen < 1),
+
+cohort as (
+SELECT
+  *
+FROM
+  last_seen_data
+WHERE
+-- filtering specifically for where the two dates match as the profile creation date sees more profiles created in days after the submission date i.e. Mar 6 has more client_ids with Mar 1 profile creation date than Mar 1
+-- Shouldn't be a material difference in the activation but should revalidate when productionizing
+  profile_date = cohort_date
+  AND submission_date = cohort_date
```

Comment:
@magorlick Need to know what you think about this. I'm specifically filtering the cohort to users who submission_date = profile_date. This is to remove some of the profile_date noise and keep the cohorts steady. I was seeing users with a Mar 1 Profile date as early as Feb 15th. I realize it's not ideal but I don't think it changes the number in a material way.

Reply:
I think this sounds good.